src/format.ts

```diff
@@ -104,6 +104,32 @@ export function automaticResultTable(results: ResultStats[]): AutomaticResults {
   return {fixed: fixedTable, unfixed: unfixedTable};
 }
 
+const nameAndMeas = (resultName: string) => {
+  const match = /(.*) \[(.*)\]/.exec(resultName);
```

Comment:
This is arguably slightly hinky. I could use the `measurementName()` helper to re-create the measurement name based on the original config, but wasn't sure it was worth it.